docs/classic.md

```diff
@@ -0,0 +1,1790 @@
+<!-- links -->
+[devforum]: https://devforum.okta.com/
+[lang-landing]: https://developer.okta.com/code/javascript
+[github-issues]: https://github.com/okta/okta-signin-widget/issues
+[deployment-checklist]: https://developer.okta.com/docs/guides/production-deployment/deployment-checklist/
+[organization]: https://developer.okta.com/docs/concepts/okta-organizations/
+[custom domain]: https://developer.okta.com/docs/guides/custom-url-domain/overview/
+[customize the template]: https://developer.okta.com/docs/guides/style-the-widget/style-okta-hosted/
+[gallery]: https://developer.okta.com/login-widget-gallery/
+[authentication]: https://developer.okta.com/docs/concepts/authentication/
+[OIDC]: https://developer.okta.com/docs/reference/api/oidc/
+[identity providers]: https://developer.okta.com/docs/concepts/identity-providers/
+[session cookie]: https://developer.okta.com/docs/guides/session-cookie/overview/
+[hosted flow]: https://developer.okta.com/docs/concepts/okta-hosted-flows/
+[redirect to a sign-in page]: https://developer.okta.com/docs/guides/sign-into-web-app/go/redirect-to-sign-in/
+[callback]: https://developer.okta.com/docs/guides/sign-into-web-app/go/define-callback/
+[authorization code flow]: https://developer.okta.com/docs/concepts/oauth-openid/#authorization-code-flow
+[PKCE]: https://developer.okta.com/docs/concepts/oauth-openid/#authorization-code-flow-with-pkce
+[implicit flow]: https://developer.okta.com/docs/concepts/oauth-openid/#implicit-flow
+[OAuth]: https://developer.okta.com/docs/concepts/oauth-openid/
+[Custom Authorization Server]: https://developer.okta.com/docs/guides/customize-authz-server/overview/
+[Authorization Server]: https://developer.okta.com/docs/concepts/auth-servers/#which-authorization-server-should-you-use
+[Social Login]: https://developer.okta.com/docs/concepts/social-login/
+[Identity Engine]: https://developer.okta.com/docs/guides/oie-intro/
+[interaction code]: https://developer.okta.com/docs/concepts/interaction-code/
+<!-- end links -->
+
+<!-- omit in toc -->
+# Okta Sign-in Widget on the Classic Pipeline
+
+The "classic" pipeline (also known as authn or V1) has been superceded by the Okta [Identity Engine][] (OIE) and the [interaction code][] flow.
+The Okta [Identity Engine][] (OIE) is a platform service that allows enterprises to build more flexible access experiences that are tailored to their organizational needs. The Okta Sign-in Widget supports OIE in all [usage](#usage-guide) scenarios. We recommend that all customers upgrade to OIE to take advantage of the expanded features and enhanced security that OIE offers. Up-to-date information on using the widget with OIE can be found in [README.md](https://github.com/okta/okta-signin-widget/blob/master/README.md)
```

Comment:
Are all customers already eligible to upgrade to OIE? Can we double check with PM please?